### PR TITLE
Condense the js

### PR DESCRIPTION
**Pull request by zoracon**
We can discuss my proposed solution but a few comments.

Always visit the principle of D.R.Y. (Don't Repeat Yourself).

I did something on line 13 to make it so you don't have to write out the push for each button each time. 

We can talk about this in particular.

### CODE REVIEW

File: calculator-project/calculator.js [hunk 1]
```diff
@@ -1,133 +1,57 @@
...
-         }
-      }
+  // creating array that stores the numbers from button clicks
+  var resultArray = [];
+  //stores numbers from resultArray that are on the left of an oppertor
+  var num1;
+  //stores numbers from resultArray that are on the right of an oppertor
+  var num2;
+  //stores the result of the calculation
+  var calculation;
```
Comment (zoracon) [inline calculator-project/calculator.js:10]: @zpaterson You can also condense empty variable declaration:
```var num1, num2, calculation;```

File: calculator-project/calculator.js [hunk 2]
```diff
@@ -1,133 +1,57 @@
...
+
+  $(".btn").click(function() {
```
Comment (zoracon) [inline calculator-project/calculator.js:12]: Study event propagation and bubbling here:
https://www.sitepoint.com/event-bubbling-javascript/

File: calculator-project/calculator.js [hunk 3]
```diff
@@ -1,133 +1,57 @@
...
+  function calculate() {
+
+    for( let i = 0; i < resultArray.length; i++) {
+
+      if(isNaN(parseInt(resultArray[i])) && (resultArray[i] !== ".") && (i !== resultArray.length -1 )) {
+          var operator = resultArray[i];
+          num1 = resultArray.splice(0, i).join("");
+          num2 = resultArray.splice(1, resultArray.length - 2).join("");
+
+          if( operator === "+") {
```
Comment (zoracon) [inline calculator-project/calculator.js:32]: Look into `switch` conditional statement as a replacement for this if statement
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

File: calculator-project/calculator.js (hunk 2 above)
Comment (zpaterson) [reply calculator-project/calculator.js:12]: Thanks for sharing this article Alexis, I'll check it out!

**Pull request closed without merging**